src/openfl/display/Bitmap.hx

```diff
@@ -77,16 +91,11 @@ class Bitmap extends DisplayObject {
 		
 	}
 	
-	private override function __enterFrame (deltaTime:Int):Void {
+	inline function __syncImageVersion (version:Int) {
 		
-		if (__bitmapData != null && __bitmapData.image != null) {
-			
-			var image = __bitmapData.image;
-			if (__bitmapData.image.version != __imageVersion) {
-				__setRenderDirty ();
-				__imageVersion = image.version;
-			}
-			
+		if (__imageVersion != version) {
+			__setRenderDirty ();
+			__imageVersion = version;
```

Comment:
Do we still need the `__imageVersion`?

Reply:
I have to think about this, this PR is here more to demonstrate the general idea :)

Reply:
Also it just occured to me that `Tilemap`/`Tileset` needs this kind of invalidation as well, ugh... Let's just remove it \o/

Reply:
yeah i think we don't need `__imageVersion`. removed.